vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -85,6 +86,19 @@ class ImageShadowNode : public ShadowNodeBase {
 
       imageViewManager->EmitImageEvent(reactImage.as<winrt::Grid>(), succeeded ? "topLoad" : "topError", source);
       imageViewManager->EmitImageEvent(reactImage.as<winrt::Grid>(), "topLoadEnd", source);
+
+
+      // ImageView gets flipped by yoga layout in RTL mode. Use ScaleTransfrom to flip it back.
+      const auto layoutDirection = winrt::Windows::ApplicationModel::Resources::Core::ResourceContext()
+                             .GetForCurrentView()
+                             .QualifierValues()
+                             .Lookup(L"LayoutDirection");
+      if (layoutDirection != L"LTR") {
+        const winrt::ScaleTransform horizontalFlipTransform;
+        horizontalFlipTransform.ScaleX(-1);
+        reactImage->RenderTransformOrigin(winrt::Point(0.5, 0.5));
+        reactImage->RenderTransform(horizontalFlipTransform);
```

Comment:
Using these properties will collide with the transform property in two ways:  1) There is a transform property that can be set on any component.  Setting a transform plus doing this invert need to be combined.  2) The transform properties internally uses "facades", which puts the UIElement into "strict" mode.  See FrameworkElementViewManager::updateProperties() for the non-animating case.  There is also an animating case to consider.  I think the solution for this will need to use facades when available.